cabra/src/pages/components/CardField.tsx

```diff
@@ -0,0 +1,35 @@
+import { ReactNode } from "react";
+import styled from "@emotion/styled";
+import tw from "twin.macro";
+
+type Span = 1 | 2 | 3;
+type Align = "left" | "center" | "right";
+
+const Title = styled.div`
+  ${tw`text-xs font-extrabold text-left`}
+`;
+
+const Field = styled.div<{ $span: Span; $align: Align }>`
+  ${tw`bg-secondary rounded-lg p-2`}
+  ${({ $span, $align }) => `
+    grid-column: span ${$span || 1} / span ${$span || 1};
+    text-align: ${$align};
```

Comment:
IMO it's a bit nicer to provide the default values when destructuring:
```suggestion
  ${({ $span = 1, $align = "left" }) => `
    grid-column: span ${$span} / span ${$span};
    text-align: ${$align};
```
It makes it harder to make the mistake of falling back to the default when a falsy value like `0` can be passed, not the case here though.
Also based on the type, you don;t even need them as `$span` and `$align` can't be undefined, but it might actually be better to have the defaults here instead having them in the `CardField` too.